Loop/Views/SettingsView.swift

```diff
@@ -197,7 +201,7 @@ extension SettingsView {
     }
     
     private var servicesSection: some View {
-        Section(header: SectionHeader(label: NSLocalizedString("Accounts", comment: "The title of the accounts section in settings"))) {
+        Section(header: SectionHeader(label: NSLocalizedString("Services", comment: "The title of the services section in settings"))) {
```

Comment:
I got the word "Accounts" from Design.  When did this change?

Reply:
Tidepool Loop isn't going to have "Services" or "Accounts", ultimately, was how I understood the plan forward. This is placeholder design for Tidepool, but it's functional in DIY.  In DIY main service we set up, NightScout, doesn't have any notion of "Accounts", and it's confusing to put it under that heading.